modules/calc/src/test/java/com/opengamma/strata/calc/config/pricing/PricingRuleTest.java

```diff
@@ -30,36 +31,32 @@
 @Test
 public class PricingRuleTest {
 
-  private static final Measure MEASURE1 = Measure.of("1");
-  private static final Measure MEASURE2 = Measure.of("2");
-  private static final Measure MEASURE3 = Measure.of("3");
-
   private static final FunctionGroup<TestTrade1> GROUP =
       DefaultFunctionGroup.builder(TestTrade1.class)
           .name("GroupName")
-          .addFunction(MEASURE1, FunctionConfig.of(TestFunction1.class))
-          .addFunction(MEASURE2, FunctionConfig.of(TestFunction2.class))
+          .addFunction(Measures.PRESENT_VALUE, FunctionConfig.of(TestFunction1.class))
```

Comment:
Unfortunate that the static imports have been lost. Must also have left the file with an unused static import. Comment applies to numerous files.